docs/buildh.md

```diff
@@ -35,11 +53,19 @@ optional arguments:
   -o OUT, --out OUT     Output base name for storing order parameters.
                         Extention ".out" will be automatically added. Default
                         name is OP_buildH.out.
+  -b BEGIN, --begin BEGIN
+                        The first frame (ps) to read from the trajectory.
+  -e END, --end END     The last frame (ps) to read from the trajectory.
+  -pi PICKLE, --pickle PICKLE
+                        Output pickle filename. The structure pickled is a dictonnary containing for each Order parameter,
+                        the value of each lipid and each frame as a matric
+
+The list of supported lipids (-l option) are: CHARMM_POPC, Berger_POPC, Berger_PLA, Berger_POP.
 ```
 
-The program needs two mandatory files (present in this repo):
-- `dic_lipids.py` (option `-l`, present in this repo) ;
-- `order_parameter_definitions_MODEL_Berger_POPC.def` (option `-d`, present in this repo).
+
+The program needs one mandatory file (present in this repo):
+- `order_parameter_definitions_MODEL_Berger_POPC.def` (option `-d`).
 
 
 #### dic_lipids.py
```

Comment:
could be removed

Reply:
Should I remove these 2 lines?
```
The program needs one mandatory file (present in this repo):
- `order_parameter_definitions_MODEL_Berger_POPC.def` (option `-d`).
```

Reply:
The interest peak is confusing. I meant : "#### dic_lipids.py"

Reply:
OK. I'll remove the corresponding paragraph.